doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -479,10 +479,17 @@ SMB Fields
 * "fuid" (string): SMB2+ file GUID. SMB1 FID as hex.
 * "share" (string): share name.
 * "share_type" (string): FILE, PIPE, PRINT or unknown.
+* "client_dialects" (array of strings): list of SMB dialects the client speaks.
+* "client_guid" (string): client GUID
+* "server_guid" (string): server GUID
+* "request.native_os" (string): SMB1 native OS string"
+* "request.native_lm" (string): SMB1 native Lan Manager string"
+* "response.native_os" (string): SMB1 native OS string"
+* "response.native_lm" (string): SMB1 native Lan Manager string"
```

Comment:
in the last 4 lines are " at the end which are unnecessary, also a mix of ending with a dot and without one :)